packages/lib/src/vis/matrix/StickyGrid.tsx

```diff
@@ -27,26 +27,36 @@ const StickyGrid = forwardRef<HTMLDivElement, Props>((props, ref) => {
     <div
       ref={ref}
       className={styles.stickyGrid}
-      style={style}
+      style={{
+        ...style,
+        width: (style.width as number) + rowHeaderCellsWidth, // account for row header cells
+        height: (style.height as number) + cellSize.height, // account for column header cells
+      }}
       role="table"
       aria-rowcount={rowCount}
       aria-colcount={columnCount}
       data-sticky={sticky || undefined}
     >
       <div className={styles.colHeaders}>
-        <div className={styles.topLeftCell} style={cellSize} data-bg />
+        <div
+          className={styles.topLeftCell}
+          style={{ width: rowHeaderCellsWidth, height: cellSize.height }}
+          data-bg
+          aria-hidden="true"
```

Comment:
Hiding header cells from the accessibility tree (`aria-rowindex`/`aria-colindex` on content cells should suffice).